README.md

```diff
@@ -32,6 +33,8 @@ This package incorporates generative AI [orchestration](https://help.sap.com/doc
 $ npm install @sap-ai-sdk/orchestration
 ```
 
+For details on Orchestration client usage, refer to [here](./packages/orchestration/README.md).
```

Comment:
[req] The phrase "refer to here" is awkward and I think wrong. Please rephrase.

Reply:
Was actually changed as part of a [review comment](https://github.com/SAP/ai-sdk-js/pull/159#discussion_r1771333094). 
Was originally `refer here`, will check what is correct gramatically.

Reply:
Done.